CodeMaid.config

```diff
@@ -22,6 +22,10 @@
       <setting name="Feature_SettingCleanupOnSave" serializeAs="String">
         <value>True</value>
       </setting>
+      <setting name="Cleaning_UpdateFileHeaderCSharp" serializeAs="String">
```

Comment:
Lets take this setting override back out please.  I get the idea of not wanting to add file headers for C# and explicitly specifying it here so user preferences weren't inherited.. but that logic would extend to all 160+ settings so I'd prefer to keep it minimal.

Reply:
Although looking back over it.. there's a couple other old ones that should really be removed too.  Either way is fine, I'll clean up this file next time around.